src/widgets/DataTable/DataTableHeaderRow.js

```diff
@@ -22,37 +22,40 @@ import { newDataTableStyles } from '../../globalStyles';
  *
  *
  */
-const DataTableHeaderRow = React.memo(({ columns, sortBy, isAscending, dispatch, sortAction }) => (
-  <HeaderRow
-    style={newDataTableStyles.headerRow}
-    renderCells={() =>
-      columns.map(({ key, title, sortable, width, alignText }, index) => {
-        const sortDirection = isAscending ? 'ASC' : 'DESC';
-        const directionForThisColumn = key === sortBy ? sortDirection : null;
-        const isLastCell = index === columns.length - 1;
-        const { headerCells, cellText } = newDataTableStyles;
-        return (
-          <HeaderCell
-            key={key}
-            title={title}
-            SortAscComponent={SortAscIcon}
-            SortDescComponent={SortDescIcon}
-            SortNeutralComponent={SortNeutralIcon}
-            columnKey={key}
-            onPressAction={sortable ? sortAction : null}
-            dispatch={dispatch}
-            sortDirection={directionForThisColumn}
-            sortable={sortable}
-            width={width}
-            containerStyle={headerCells[alignText || 'left']}
-            textStyle={cellText[alignText || 'left']}
-            isLastCell={isLastCell}
-          />
-        );
-      })
-    }
-  />
-));
+const DataTableHeaderRow = React.memo(({ columns, sortBy, isAscending, dispatch, sortAction }) => {
+  const { headerRow, headerCells, cellText } = dataTableStyles;
+  return (
+    <HeaderRow
+      style={headerRow}
+      renderCells={() =>
+        columns.map(({ key, title, sortable, width, alignText }, index) => {
+          const sortDirection = isAscending ? 'ASC' : 'DESC';
```

Comment:
The usual nit: can these be made into constants?

Reply:
this PR is for moving files, not making code changes

Reply:
Doesn't have to be done here? I think a good practice (what we usually do?) is to suggest making a new issue for changes that don't fit within the remit of a PR (or dismiss if you don't think worth doing).

Reply:
sure no problem - I will probably not make a new PR and just sneak it into some other PR, or you can !